src/views/layout/header.ejs

```diff
@@ -24,11 +24,11 @@
         </ul>
     </nav>
     <div class="language-picker mr-2">
-        <a class="cursor-pointer nav-link pr-4 mr-0 lang-trigger"><%= currentLanguage.toUpperCase() %></a>
+        <a class="cursor-pointer nav-link pr-4 mr-0 lang-trigger"><span class="flag-icon flag-icon-<%= currentLanguage == 'en' ? 'gb' : currentLanguage %>"></span></a>
```

Comment:
We will have the same problem for danish, right?

Reply:
Yep, that "fix" was more quick and dirty, I thought about a assiocative array which has all "problematic" languages listened, an then do a check like myassio[language] ? myassio[language] : language but im not shure how javascript handles undefined indexes